### DISCUSSION TITLE
Managing New-DiagramNode settings

### DISCUSSION THREAD

**Discussion by kwein1** [EvotecIT/PSWriteHTML#261] [Q&A]
Using the following code, I'm able to create a hierarchical "tree" of bubbles" from my data.
```
	New-DiagramOptionsLayout -HierarchicalEnabled $true
				New-DiagramOptionsPhysics -Enabled $true -HierarchicalRepulsionAvoidOverlap 1 -HierarchicalRepulsionNodeDistance 100


```
My nodes that are single words show up nice and small. My multi-word ones streeeeeetch way too far right-left,
![2021-08-16 10_10_39-Scheduled Tasks](https://user-images.githubusercontent.com/4205035/129577683-5d948478-e93a-46f4-b7bd-947edc47bcbf.png)

So I decided to break them internally, to make them taller and skinnier. But then it creates lots of white space around them.
```
	$UseLabel = $TreeNode.Name -replace " ", "`r`n"
	# For some reason, multi-line labels create too much white space inside the node bubble in HTML - how to reduce that
	#  to be more like single line?
	New-DiagramNode -Label $UseLabel -id "$($TreeNode.NodeId)" -FontSize 40 -level $UseLevel -To $TreeNode.ParentNodeId
```
How can I tighten up the white space around the multi-line text? (I'll try to attach a screenshot to make this painfully obvious.) 
![Balloon](https://user-images.githubusercontent.com/4205035/129577584-eb5c9b10-f4c1-4377-b63a-ced77ae686dd.png)

**Comment by PrzemyslawKlys** [comment]
I would probably pick an icon/image as the DiagramNode and have text below it with proper break lines.

**Reply by kwein1** [reply]
Do most people do that via downloading a font library like FontAwesome and then using a line like:
New-DiagramNode -Label 'Internet' -Image 'https://cdn.iconscout.com/icon/premium/png-64-thumb/internet-2102080-1768387.png' -ImageType squareImage -Id 'Internet1' -Level 2
?

**Reply by PrzemyslawKlys** [reply]
There are multiple ways to deal with this. I can't tell what everyone else is doing, but I can show you what I do. And FontAwesome is built-in into New-DiagramNode. You can use IconSolid, IconRegular, or IconBrands.

I recommend: 
- https://evotec.xyz/creating-office-365-migration-diagram-with-powershell/
- https://evotec.xyz/easy-way-to-create-diagrams-using-powershell-and-pswritehtml/

And generally, there are links to other articles on evotec which show how you can make the best use case of PSWriteHTML. Or just go thru examples on GitHub.
